hail/python/hail/docs/change_log.md

```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## 0.2.21
+
+Release 2019-08-30
+
+### Correctness
+- (hail#6945) Fix `expand_types` to preserve the order of the data in a table
+- (hail#6958) Fix stack overflow errors when counting the result of a `Table.union`
+
+### Functionality
+- (hail#6913) Teach `sparse_split_multi` how to filter changed loci
```

Comment:
sparse_split_multi is experimental and not really publicly exposed, I'd remove this.